kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -169,7 +175,7 @@ private void initialize() {
 
         fieldsTilePane.getChildren().clear();
         fieldsTilePane.setPrefColumns(2);
-
+        otherNamesVBox.getChildren().clear();
```

Comment:
Should we call `clearView()`? 
Not sure if there a method called `clearView()` that could be called?
After closing the Panel, clearView() could be called again.

Reply:
There is no method called clearView this is not an input form controller but details view controller.